### PR TITLE
Nontynet : bio added

### PR DESCRIPTION
**Pull request by mintoming**
catalog/designers/nontynet

### CODE REVIEW

File: catalog/designers/nontynet/info.pb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+designer: "Nontynet"
```
Comment (davelab6) [inline catalog/designers/nontynet/info.pb:1]: I'd prefer `Nonty` to this, what do you think @mintoming ?

Comment (mintoming) [reply catalog/designers/nontynet/info.pb]: Name change pushed below :)

**Comment by mintoming** [comment]
@m4rc1e @davelab6 could you help me on how to fix this fail? 🥺🙏

**Comment by m4rc1e** [comment]
@mintoming LGTM.

Don't worry about that fail. It's failing because you've touched a font project which requires certain tooling that you don't have credentials for.

**Pull request merged**